apps/cli/src/main.ts

```diff
@@ -107,7 +109,11 @@ program
 
         if (!options.templateType) throw new Error('Template type is not set')
         if (!options.featureBoardKey && !bearerToken)
-            throw new Error('Feature Board Key is not set')
+            throw new Error(
+                options.nonInteractive
+                    ? 'Feature Board Key is not set'
```

Comment:
FeatureBoard

Reply:
Updated all "feature boards"'s to FeatureBoard